modular/stonekeep/code/jobs/job_types/adventurer/types/combat/barbarian.dm

```diff
@@ -52,13 +52,13 @@
 	switch(weapontype)
 		if("Sword")
 			beltr = /obj/item/weapon/sword/iron
-			H.mind?.adjust_skillrank(/datum/skill/combat/swords, 2, TRUE)
+			H.mind?.adjust_skillrank(/datum/skill/combat/swords, 1, TRUE)//to guarantee average skill on a weapon + skilled in another
```

Comment:
See, just like how it's done here, do the same on Amazon: Grant her skill levels on the weapon(s) of their choice.